dist/images/ovnkube.sh

```diff
@@ -1213,7 +1213,7 @@ ovs-metrics() {
 }
 
 etcd () {
-  if [[ ${ovsdb_etcd_log_to_tcpdump} == "true" ]]; then
+  if [[ ${ovndb_etcd_tcpdump} == "true" ]]; then
```

Comment:
should we compare it wuth true? cannot it be without ("== "true" ) ?

Reply:
I want to stay consistent with all the other comparisons on the `ovnkube.sh` script.
for example 
on 862:
```
if [[ ${ovn_hybrid_overlay_enable} == "true" ]]; then
```
on 869
```
if [[ ${ovn_disable_snat_multiple_gws} == "true" ]]; then
```
and so forth..
moreover as seen [https://unix.stackexchange.com/questions/105935/understanding-boolean-operators-in-bash-script](url):

```
Bash variables don't have types, so there's no such thing as a boolean variable or value like true or false. Basically all bash variables are just strings.
```
so as far as I understand , the `[[ VAR ]]` will be false only if it will non empty string.
so we can't give up the "("== "true" )" part.
here is a quick demo:
![image](https://user-images.githubusercontent.com/41959855/121366055-d5165580-c941-11eb-9b79-70cca35e646f.png)